sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/cases/EditCasePage.java

```diff
@@ -178,4 +178,8 @@
   public static final By CREATE_NEW_PERSON_CHECKBOX =
       By.xpath("//label[text()='Create a new person']");
   public static final By CREATE_NEW_CASE_CHECKBOX = By.xpath("//label[text()='Create a new case']");
+  public static final By CURRENT_HOSPITALIZATION_POPUP =
+      By.xpath("//*[contains(text(),'Current hospitalization')]");
```

Comment:
cannot be taken by other identifier? to make it reusable also for DE if needed

Reply:
same as above